conf/messages

```diff
@@ -231,6 +231,11 @@ annotation.reopen.tooLate=The annotation cannot be reopened anymore, since it ha
 annotation.reopen.notAllowed=You are not allowed to reopen this annotation.
 annotation.reopen.notFinished=The requested annotation is not finished.
 annotation.reopen.failed=Failed to reopen the annotation.
+annotation.reopen.locked=This annotation is locked by the owner and therefore cannot be reopened.
+annotation.isLockedByUser.notAllowed=Only the owner of this annotation is allowed to change the locked state of an annotation.
```

Comment:
not even an admin can do so?

Reply:
I think this is what the current implementation does, yes. We should discuss it. Would you prefer if admins were allowed to lock/unlock other people’s annotations? I would vote for it. cc @normanrz

Reply:
> Would you prefer if admins were allowed to lock/unlock other people’s annotations?

I think, I'm against that. For the same reason that only the owner can mutate an annotation.. If we should decide to allow admins this, then, the interface should make it very clear that one is locking/unlocking somebody else's annotation.

Reply:
Fine by me too. Restricting it to the owner certainly is easier for the moment

Reply:
So, if a user leaves the team, an admin would need to copy the annotation to make it editable again?

Reply:
> So, if a user leaves the team, an admin would need to copy the annotation to make it editable again?

In your described scenario -> yes

Reply:
After we agreed upon this, this pr should (and its potential changes) this pr should be ready for the 2nd review round

Reply:
I just talked with @fm3 about this pending decision: -> Should we decided that admins can also unlock/lock an annotation, this can be done as a follow-up pr :)